docs/2.7.0/versions.json

```diff
@@ -1,5 +1,6 @@
 {
   "daml": "2.7.0-snapshot.20230523.11810.0.06abd5c6",
   "canton": "2.7.0-snapshot.20230623.10628.0.v51275048",
-  "daml_finance": "1.2.14.99.20230615.1"
+  "daml_finance": "1.2.14.99.20230615.1",
+  "canton_drivers": "0.1.9"
```

Comment:
This downloads about 292 MB while the to be included files are just a few kilobytes.

I recommend to create a docs-only assembly tarball which only contains what the docs.daml.com documentation build needs to build.

Reply:
i have updated the tar ball we generate, once we do the next release it should only contain relevant files